src/components/JobOpenings.js

```diff
@@ -304,21 +299,20 @@ const JobOpenings = () => (
             delivery pipelines.
           </li>
           <li>
-            Previous experience contributing or managing open source projects.
+            Previous experience contributing or managing open source projects and setting up nodes.
```

Comment:
"setting up nodes" is pretty generic; we may want to use another bullet point to just say "setting up decentralized software, e.g. Ethereum and IPFS nodes"